README.md

```diff
@@ -6,6 +6,23 @@ REST APIs using Jersey, Jackson, Jetty, and Hibernate Validator.
 
 See the [`examples/`](examples/) directory for a simple demo service.
 
+Security
+--------
+The REST `Application` can support both http and https. https is disabled by default. Three modes are supported:
+http, https, and http+https. In the last mode, http and https are supported on different ports.
+
+The https implementation is similar to Kafka, where the server specifies a keystore and trust store. When SSL
+client auth is configured, the client must authenticate -- the key in the client's keystore must be trusted by
+the server's trust store, either because the client cert is in the server's trust store, or because the CA that signed
+the client's cert is in the server's trust store.
+
+Metrics
+-------
+When both http and https protocols are enabled, two metrics groups are created, `jetty-metrics` and
+`jetty-https-metrics`. However, `jetty-https-metrics` is a duplicate of `jetty-metrics`. Put differently, each
+metrics group will have the same value for all metrics, and those values will be influenced by both http and
```

Comment:
Currently, `io.confluent.common.metrics` does not support registering multiple metrics listeners with the same name. As such, there are a few options:

 1. My current implementation, where two different groups exist, but they have identical values.
 2. Change `io.confluent.common.metrics` to support adding multiple listeners with the same group name.
 3. Use a different group name, and different metrics tags, which means https and http would have different values.

I chose (1) because if I were operating a rest app, I would want to see the same metrics for http and https, yet I thought the change to `io.confluent.common.metrics` may be problematic because it would the current semantics quite a bit. Although, thinking about this more, I may actually prefer http and https values to be different, so get more monitoring granularity.

I agree (1) is odd and am happy to consider one of the other alternatives. I'm leaning towards option (3). I'd love to hear your thoughts on options (2) and (3).